activerecord/CHANGELOG.md

```diff
@@ -38,7 +38,6 @@
 
     ```ruby
     class User < ApplicationRecord
-      has_many :bookmarks
```

Comment:
The association was used below to demonstrate usage of strict_loading on a Model. Is the removal intentional or removed by mistake while rebasing the PR?

Reply:
mistake, adding back

Reply:
Raised a PR for it: https://github.com/rails/rails/pull/41329. Please feel free to close it if you have added the change back 😊